openshift/templates/wfwx_hourly_actuals.cronjob.yaml

```diff
@@ -20,7 +20,7 @@ parameters:
   - name: IMAGE_REGISTRY
     value: image-registry.openshift-image-registry.svc:5000
   - name: JOB_NAME
-    value: bcfw-p1-hourly-actuals
+    value: wfwx-hourly-actuals
```

Comment:
Remember to take care when pushing to production to manually delete bcfw-p1-hourly-actuals-wps-prod